dimes/dimensional_plot.py

```diff
@@ -227,6 +227,8 @@ def __init__(
         x_axis: DimensionalData | TimeSeriesAxis | list[SupportsFloat] | list[datetime],
         title: str | None = None,
         additional_info: str | None = None,
+        width: int | None = None,
+        height: int | None = None,
```

Comment:
It looks like these are never set in any calls to this function. Can we remove them?